app/views/profiles/update.js.erb

```diff
@@ -1,3 +1,15 @@
-var button = $("#toggle-admin-emails");
-var newText = "<%= @user.admin_email_delivery ? "On <i class='fas fa-fw fa-toggle-on'></i>".html_safe : "Off <i class='fas fa-fw fa-toggle-off'></i>".html_safe %>";
-button.html(newText);
\ No newline at end of file
+if (<%= @notice.present? %>) {
+  // modal unnecessary as toggle has already changed
+  console.log('<%= @notice %>')
+} else {
```

Comment:
Simpler, with less entangled JS and Ruby, and no unnecessary debug logging:

```suggestion
const isNoticePresent = <%= @notice.present? %>
if (!isNoticePresent) {
```

Reply:
This is actually inverted from what we want - it should be `if (ifNoticePresent)`

Reply:
It looks non-inverted to me -- note how `if (!isNoticePresent) {` in the suggestion is where the `else` is in the original.  Omitting the original `if` clause also leaves out the unnecessary `console.log`, per SCP convention.